src/views/List.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="container">
+    <template v-if="appStatus === 'loading'">
+      <LoadingSpinner />
+    </template>
+    <template v-else>
+      <h1>{{ list.title }}</h1>
+    <promises-table :promises="promises" exclude="[source_name, politician_name]" />
+
+    </template>
+  </div>
+</template>
+
```

Comment:
same here

Reply:
ok

Reply:
this one serves a purpose thought. <template> and <script> in Vue's single component files are separate sections completely.